application/build.gradle

```diff
@@ -45,7 +45,7 @@ dependencies {
     implementation 'org.apache.logging.log4j:log4j-api:2.14.1'
     implementation 'org.apache.logging.log4j:log4j-core:2.14.1'
     implementation 'org.apache.logging.log4j:log4j-slf4j18-impl:2.14.1'
-    implementation 'org.jooq:jooq:3.15.2'
+    implementation 'org.jooq:jooq:3.15.3'
```

Comment:
Note: We can probably remove this explicit dependency and let the `database` dependency handle that?